<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindSubtitlesConfigDialog</class>
 <widget class="QDialog" name="FindSubtitlesConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Advanced options</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Server</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&amp;Opensubtitles server:</string>
        </property>
        <property name="buddy">
         <cstring>server_edit</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="server_edit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Proxy</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QCheckBox" name="use_proxy_check">
        <property name="text">
         <string>&amp;Enable proxy</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="proxy_container" native="true">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>&amp;Host:</string>
           </property>
           <property name="buddy">
            <cstring>proxy_hostname_edit</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QLineEdit" name="proxy_hostname_edit"/>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>&amp;Port:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="buddy">
            <cstring>proxy_port_spin</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QSpinBox" name="proxy_port_spin">
           <property name="maximum">
            <number>65535</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>&amp;Username:</string>
           </property>
           <property name="buddy">
            <cstring>proxy_username_edit</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="4">
          <widget class="QLineEdit" name="proxy_username_edit"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Pa&amp;ssword:</string>
           </property>
           <property name="buddy">
            <cstring>proxy_password_edit</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="4">
          <widget class="QLineEdit" name="proxy_password_edit">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>&amp;Type:</string>
           </property>
           <property name="buddy">
            <cstring>proxy_type_combo</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="proxy_type_combo"/>
         </item>
         <item row="3" column="2" colspan="3">
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>261</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>server_edit</tabstop>
  <tabstop>use_proxy_check</tabstop>
  <tabstop>proxy_hostname_edit</tabstop>
  <tabstop>proxy_port_spin</tabstop>
  <tabstop>proxy_username_edit</tabstop>
  <tabstop>proxy_password_edit</tabstop>
  <tabstop>proxy_type_combo</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FindSubtitlesConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FindSubtitlesConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>use_proxy_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>proxy_container</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
